### PR TITLE
ClassComments-Via-Class 

### PR DESCRIPTION
**Pull request by MarcusDenker**
This PR changes all cliens to as the class directly for comments, not the #organizer

This prepares to move the classCommentSourcePointer to ClassDescription

This turns for now off the automatic class template, this should be done in the UI, not in the model

### CODE REVIEW

File: src/Kernel/ClassDescription.class.st [hunk 1]
```diff
@@ -334,8 +333,7 @@ ClassDescription >> classesThatImplementAllOf: selectorSet [
 
 { #category : #'accessing - comment' }
 ClassDescription >> comment [
-    "Answer the receiver's comment. (If missing, supply a template) "
-    ^self instanceSide organization classComment ifEmpty: [ self classCommentBlank ]
```
Comment (privat) [inline src/Kernel/ClassDescription.class.st:338]: yes!

**Review by privat — APPROVED** [review]

**Review by jecisc — APPROVED** [review]
I like to see all those #organization vanish :)

Do you plan to move the empty class template in the UI soon? (just wondering)

**Comment by MarcusDenker** [comment]
For the UI? yes, I will check where the browser calls #comment and do the check there

**Comment by MarcusDenker** [comment]
Even though I am not anymore that convinced about the template...

**Comment by privat** [comment]
I like the removal of the template, in a model POV I think that "no comment" is important information to preserve.
The template should be used when developers want to comment the class: i.e. not on default presentation, but when entering in edition mode.

**Pull request merged**